backend/controllers/passController.js

```diff
@@ -44,39 +52,30 @@ const viewPasses = async (req, res, next) => {
   }
 }
 
-//@desc View unverified passes to the gaurd
-//@route GET /passes/viewUnverifiedPass
-const viewUnverifiedPasses = async (req, res, next) => {
-  try {
-    const viewUnverified = await Pass.find({ checkInTime: null })
-    res.status(201).json({
-      success: true,
-      viewUnverified: viewUnverified,
-      message: "Unverified Passes fetched successfully",
-    })
-  } catch (err) {
-    next(err)
-  }
-}
-
 //@desc Verify pass by guard
-//@route PUT /users/verifyPass
+//@route PATCH /users/verifyPass
 const verifyPass = async (req, res, next) => {
   try {
-    const passVerified = await Pass.findOneAndUpdate(
-      { _id: req.params._id },
-      {
-        checkInTime: new Date(),
-      }
-    )
-    res.status(201).json({
-      success: true,
-      passVerified: passVerified,
-      message: "Pass verified successfully",
-    })
+    const { otp } = req.body
+    if (!otp) throw new ErrorHandler(400, "OTP is required")
+    const findOTP = await Pass.findOne({ OTP: otp })
+    console.log(findOTP, otp, typeof otp)
+    if (!findOTP) throw new ErrorHandler(404, "Invalid OTP")
+    else if (findOTP) {
```

Comment:
Is else if really required?

Reply:
no...working without it as well, so removing it